src/ffi/dictobject.rs

```diff
@@ -92,6 +92,14 @@ extern "C" {
         key: *mut *mut PyObject,
         value: *mut *mut PyObject,
     ) -> c_int;
+    #[cfg(not(PyPy))]
+    pub fn _PyDict_Next(
```

Comment:
https://github.com/python/cpython/blob/a24107b04c1277e3c1105f98aff5bfa3a98b33a0/Include/cpython/dictobject.h#L49 and elsewhere for the others.